plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -60,6 +60,8 @@ class SlurmQueueConf(BaseQueueConf):
     # Useful to add parameters which are not currently available in the plugin.
     # Eg: {"mail-user": "blublu@fb.com", "mail-type": "BEGIN"}
     additional_parameters: Dict[str, Any] = field(default_factory=dict)
+    # Maximum number of jobs running in parallel
+    array_parallelism: int = 256
```

Comment:
short and sweet.
Anything else we should be adding from the submitit API while we are here?

Reply:
Good question, none that I needed at least, since `additional_parameters` allows to pass parameters directly to `slurm`.
Yet, some parameters from [submitit API](https://github.com/facebookincubator/submitit/blob/abf1993cef94b84c480fc0d7dc7676700996e700/submitit/slurm/slurm.py#L387) are missing from the [hydra plugin API](https://github.com/facebookresearch/hydra/blob/affb12a85c64bce17408aab890ff6de30293ff3a/plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py) like `cpus_per_gpu` but I'm only starting to use slurm so no idea whether that is useful.

Reply:
@jrapin, in your opinion - anything missing that we should add while we are at it?

Reply:
I think only the variables [here](https://github.com/facebookincubator/submitit/blob/abf1993cef94b84c480fc0d7dc7676700996e700/submitit/slurm/slurm.py#L443) should be considered (since everything else can be passed via `additional_parameters`). I think `map_count` could be useful.

Reply:
what is it useful for?

Reply:
It decides the number of jobs to be run in parallel. Digging more, it does not need to be set explicitly.

Reply:
> @jrapin, in your opinion - anything missing that we should add while we are at it?

sorry for the late reply :s
I can't think of anything

Reply:
No worries, thanks @jrapin !